test/test_nn.py

```diff
@@ -13907,6 +13837,81 @@ def helper(n, c, h, w, ks):
         helper(2, 8, 4, 4, ks=2)
         helper(None, 3, 50, 50, ks=5)
 
+    def test_upsamplingNearest2d(self, device):
```

Comment:
were there any changes to this test?

Reply:
It is moved to "device-generic" test (if that's the correct name). Previously it only takes `self`, now it takes `(self, device)`, so that we can test the functionalities on CUDA as well.